### PR TITLE
fix(NcAppNavigationCaption ): properly align with NcAppNavigationItem

### PR DESCRIPTION
**Pull request by GVodyanov**
Make NcAppNavigationCaption properly align with NcAppNavigationItem

### ☑️ Resolves

- Fix https://github.com/nextcloud/calendar/pull/6129#issuecomment-2225327507

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
 Screenshot from 2024-07-15 10-16-03 | Screenshot from 2024-07-15 10-14-38

### CODE REVIEW

File: src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue [hunk 1]
```diff
@@ -240,7 +240,7 @@ export default {
 		box-shadow: none !important;
 		flex-shrink: 0;
 		// padding to align the name with the icon of app navigation items
```
Comment (ChristophWurst) [inline src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue:242]: this comment confirms the alignment needed adjustment ;-)

**Review by ChristophWurst — APPROVED** [review]
:sunglasses:

**Review by marcoambrosini — APPROVED** [review]

**Comment by ChristophWurst** [comment]
> Failed to fetch dynamically imported module: http://localhost:5173/__cypress/src/@fs/home/runner/work/nextcloud-vue/nextcloud-vue/cypress/component/NcSelect.cy.ts

CI job restarted. This seems unrelated.

**Comment by GVodyanov** [comment]
I changed one CSS character and CI still fails 😭

**Comment by ChristophWurst** [comment]
@ShGKme could you help finding out if force merge is ok here?

**Comment by ShGKme** [comment]
> @ShGKme could you help finding out if force merge is ok here?

Just restart CI until it's green (from 1 to 5 times)

**Comment by ShGKme** [comment]
Restarted and green now

**Comment by ShGKme** [comment]
Don't forget the checklist and bug/enhancement tags for changelog gen 👀

**Pull request merged**